### PR TITLE
1838/ami chart jurisdictionalized

### PR DESCRIPTION
**Pull request by YazeedLoonat**
## Issue Overview
Enforces Jurisdiction on the AMI charts for the partners portal

This PR addresses #issue
https://github.com/bloom-housing/bloom/issues/1838

- [x] This change addresses the issue in full

## Description
Leverages the AMI Chart -> Jurisdiction relationship to only pull back AMI Charts on the current jurisdiction

## How Can This Be Tested/Reviewed?
The Easiest way to test this is to set some AMI charts to a different jurisdiction than the one you are on
From there ensure that when you hit the partner portal -> listing form -> units section that the AMI charts listed are only for the jurisdiction you are "on"

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have assigned reviewers
- [x] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [ ] I have exported any new pieces added to ui-components
- [x] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

**Comment by YazeedLoonat** [comment]
Hey @emilyjablonski can you peep this real quick and make sure I added a detroit based ami-chart correctly please

File: backend/core/src/ami-charts/ami-charts.service.ts [hunk 1]
```diff
@@ -22,6 +22,8 @@ export class AmiChartsService {
       where: (qb) => {
         if (queryParams.jurisdictionName) {
           qb.where("jurisdiction.name = :jurisdictionName", queryParams)
+        } else if (queryParams.jurisdictionid) {
+          qb.where("jurisdiction.id = :jurisdictionid", queryParams)
```
Comment (emilyjablonski) [inline backend/core/src/ami-charts/ami-charts.service.ts:26]: NBD but everywhere else this is named `jurisdictionId`

File: backend/core/src/ami-charts/dto/ami-chart-list-query-params.ts [hunk 2]
```diff
@@ -13,4 +13,14 @@ export class AmiChartListQueryParams {
   @IsOptional({ groups: [ValidationsGroupsEnum.default] })
   @IsString({ groups: [ValidationsGroupsEnum.default] })
   jurisdictionName?: string
+
+  @Expose()
+  @ApiProperty({
+    name: "jurisdictionid",
```
Comment (emilyjablonski) [inline backend/core/src/ami-charts/dto/ami-chart-list-query-params.ts:19]: NBD but everywhere else this is named `jurisdictionId`

File: backend/core/src/seeds/listings/listing-triton-seed.ts [hunk 3]
```diff
@@ -820,3 +820,65 @@ export class ListingTritonSeed extends ListingDefaultSeed {
...
+  async seed() {
+    const unitTypeOneBdrm = await this.unitTypeRepository.findOneOrFail({ name: "oneBdrm" })
+    const unitTypeTwoBdrm = await this.unitTypeRepository.findOneOrFail({ name: "twoBdrm" })
+
+    const detroitJurisdiction = await this.jurisdictionRepository.findOneOrFail({
+      name: CountyCode.detroit,
+    })
+    const amiChart = await this.amiChartRepository.save({
+      ...tritonAmiChart,
+      name: `${tritonAmiChart.name}Detroit`,
```
Comment (emilyjablonski) [inline backend/core/src/seeds/listings/listing-triton-seed.ts:834]: NBD but this comes through as `San Jose TCAC 2019Detroit`

**Review by emilyjablonski — APPROVED** [review]
LGTM locally! Before merge can you wait for the backend tests to run and make sure any failures are unrelated?

**Pull request merged**